app/src/js/components/Collections/granules.js

```diff
@@ -42,7 +42,7 @@ const CollectionGranules = ({
   const displayName = strings.granules;
   const collectionId = getCollectionId(params);
   const view = getView();
-  const [workflow, setWorkflow] = useState();
+  const [workflow, setWorkflow] = useState(workflowOptions[0]);
```

Comment:
Need to set initial state. Execute was failing if you didn't select a new workflow from the dropdown. This allows the first value to actually be used.